apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/db/AplDbVersion.java

```diff
@@ -784,8 +784,15 @@ protected int update(int nextUpdate) {
             case 297:
                 apply("CREATE TABLE IF NOT EXISTS dex_contract (db_id IDENTITY NOT NULL, offer_id BIGINT NOT NULL, " +
                         "counter_offer_id BIGINT NOT NULL, secret_hash CHAR(64) NOT NULL, height INT NOT NULL, latest BOOLEAN NOT NULL DEFAULT TRUE)");
-            case 298:
-                return 298;
+            case 298: 
```

Comment:
That will be conflicting when stage is merged into master. Just FYI

Reply:
that's aparent. Anyway, while developing and merging sharding-related branch, that version should be updated. We will cross that bridge when it comes to it.